src/server/api/controllers/quiz-results.controller.js

```diff
@@ -24,9 +24,15 @@ const getQuizResults = async (answerId, userId) => {
         .where('fk_answer_id', '=', answerId)
         .select('id', 'fk_answer_id', 'fk_user_id', 'created_date');
     } else if (userId !== undefined) {
+      // get total number of questions from questions table.
+      const numberOfQuestions = (await knex('questions').count())[0][
+        'count(*)'
+      ];
```

Comment:
This is a bit hard to read :) and does this work? and could there maybe be a different way of writing this?

Reply:
So doesn't this return the same?
```suggestion
      const numberOfQuestions = await knex.raw('select count(*) from questions')
```

Reply:
no it does not. it returns complicated object and then we have to pick  from that object.

Reply:
[](url)
<img width="972" alt="vlcscreen" src="https://user-images.githubusercontent.com/64294952/118395399-9646f580-b64a-11eb-9e58-e3b1351d5e5d.png">